### PR TITLE
NativeMapView.addImage without copying bitmap into Java

### PR DESCRIPTION
**Pull request by tobrun**
Follow up from external PR in https://github.com/mapbox/mapbox-gl-native/pull/11000. Capturing from @ivovandongen that the config validation part should be re-added:

```java
    // Check/correct config
    if (image.getConfig() != Bitmap.Config.ARGB_8888) {
      image = image.copy(Bitmap.Config.ARGB_8888, false);
    }
```

cc @jaegs

### CODE REVIEW

**Comment by jaegs** [comment]
Maybe the validation part should go here:
https://github.com/mapbox/mapbox-gl-native/blob/4d358260140c9d52103cc95631a8519b969bc408/platform/android/src/bitmap.cpp#L112-L113

**Comment by tobrun** [comment]
@jaegs you are right, will add the conversion on the c++ side.

@ivovandongen I tested image conversion with:

```java
Bitmap bitmap = Bitmap.createBitmap(createColors(), WIDTH, HEIGHT, Bitmap.Config.RGB_565);
mapboxMap.addImage("image", bitmap);

GeoJsonSource geoJsonSource = new GeoJsonSource("source", Feature.fromGeometry(Point.fromCoordinates(new double[] {0, 0})));
mapboxMap.addSource(geoJsonSource);

SymbolLayer symbolLayer = new SymbolLayer("layer", "source");
symbolLayer.withProperties(iconImage("image"));
mapboxMap.addLayer(symbolLayer);
```

And I was able to debug that the conversion is happening as expected:

screen shot 2018-02-09 at 10 54 44

This resulted in:

screen shot 2018-02-09 at 10 56 11

File: platform/android/src/bitmap.cpp [hunk 1]
```diff
@@ -128,5 +127,12 @@ PremultipliedImage Bitmap::GetImage(jni::JNIEnv& env, jni::Object<Bitmap> bitmap
     return { Size{ info.width, info.height }, std::move(pixels) };
 }
 
+jni::Object<Bitmap> Bitmap::Copy(jni::JNIEnv& env, jni::Object<Bitmap> bitmap) {
+    using Signature = jni::Object<Bitmap>(jni::Object<Config>, jni::jboolean);
+    auto method = _class.GetMethod<Signature>(env, "copy");
```
Comment (ivovandongen) [inline platform/android/src/bitmap.cpp]: I think this should be static to cache the method lookup

**Comment by tobrun** [comment]
Could you re-review @ivovandongen? static keyword was added.

**Pull request merged**